tools/ci_build/github/azure-pipelines/stages/jobs/react-natvie-andriod-e2e-test-job.yml

```diff
@@ -0,0 +1,211 @@
+      versionSpec: '20.x'
+
+  - script: |
+      sudo apt install coreutils ninja-build nodejs npm yarn
+      npm install --global yarn
+    displayName: Install coreutils, ninja, npm, and yarn
+
+  - task: DownloadPipelineArtifact@2
+    inputs:
+      buildType: 'current'
+      artifactName: '${{parameters.ArtifactName}}'
+      targetPath: '$(Build.BinariesDirectory)/android-full-aar'
+    displayName: Download Android AAR artifacts
+
+  - task: CopyFiles@2
+    inputs:
+      sourceFolder: $(Build.BinariesDirectory)/android-full-aar
+      contents: onnxruntime-android-*.aar
+      targetFolder: $(Build.SourcesDirectory)/js/react_native/android/libs
+    displayName: Copy Android package to React Native directory
+
+  - script: |
+      npm ci
+    workingDirectory: '$(Build.SourcesDirectory)/js'
+    displayName: npm ci js
+
+  - script: |
+      npm ci
+    workingDirectory: '$(Build.SourcesDirectory)/js/common'
+    displayName: npm ci js/common
+
+  - script: |
+      yarn
+    workingDirectory: '$(Build.SourcesDirectory)/js/react_native'
+    displayName: yarn js/react_native
+
+  - task: PowerShell@2
+    inputs:
+      filePath: '$(Build.SourcesDirectory)/tools/ci_build/github/js/pack-npm-packages.ps1'
+      arguments: '"-dev.$(Get-Date -Format yyyyMMdd)-$(git rev-parse --short HEAD)" $(Build.SourcesDirectory) react_native'
+      workingDirectory: '$(Build.SourcesDirectory)'
+      errorActionPreference: stop
+    env:
+      ORT_JS_PACK_MODE: e2e
+    displayName: Pack NPM packages
+
+  - script: |
+      mv $(Build.SourcesDirectory)/js/common/onnxruntime-common*.tgz onnxruntime-common.tgz
+      yarn add --no-lockfile file:./onnxruntime-common.tgz
+      mv $(Build.SourcesDirectory)/js/react_native/onnxruntime-react-native*.tgz onnxruntime-react-native.tgz
+      yarn add --no-lockfile file:./onnxruntime-react-native.tgz
+      yarn
+    workingDirectory: '$(Build.SourcesDirectory)/js/react_native/e2e'
+    displayName: Bootstrap Android and iOS e2e tests
+
+  - script: |
+      yarn add --dev jest-junit
+    workingDirectory: '$(Build.SourcesDirectory)/js/react_native/e2e'
+    displayName: install jest junit reporter js/react_native/e2e
+
+  - script: |
+      keytool -genkey -v -keystore debug.keystore -alias androiddebugkey -storepass android \
+        -keypass android -keyalg RSA -keysize 2048 -validity 999999 -dname "CN=Android Debug,O=Android,C=US"
+    workingDirectory: '$(Build.SourcesDirectory)/js/react_native/e2e/android'
+    displayName: Generate a debug keystore
+
+  - task: CopyFiles@2
+    inputs:
+      sourceFolder: $(Build.BinariesDirectory)/android-full-aar
+      contents: onnxruntime-*.aar
+      targetFolder: $(Build.SourcesDirectory)/js/react_native/e2e/android/app/libs
+    displayName: Copy Android package to Android e2e test directory
+
+  - script: |
+      yarn global add detox-cli
+      echo "Path: $PATH"
+      echo "##vso[task.prependpath]$(yarn global bin)"
+      echo "Updated PATH: $PATH"
```

Comment:
The PATH var isn't updated until next step. See [StackOverflow](https://stackoverflow.com/questions/57322531/how-to-set-path-in-azure-pipeline-using-prependpath/57338593#57338593)